Doc/library/functions.rst

```diff
@@ -1726,44 +1726,42 @@ are always available.  They are listed here in alphabetical order.
 
    Example::
 
-      >>> for item in zip(('a', 'b', 'c'), (1, 2, 3)):
+      >>> for item in zip(['a', 'b', 'c'], [1, 2, 3]):
       ...     print(item)
       ...
       ('a', 1)
       ('b', 2)
       ('c', 3)
 
    More formally: :func:`zip` returns an iterator of tuples, where the *i*-th
-   tuple contains the *i*-th element from each of the argument sequences or
-   iterables.
+   tuple contains the *i*-th element from each of the argument iterables.
 
-   :func:`zip` returns a lazy iterable. The elements wouldn't be processed
-   until the iterable is iterated on, e.g. by wrapping in a :class:`tuple`.
+   :func:`zip` is lazy: The elements wouldn't be processed until the iterable
+   is iterated on, e.g. by wrapping in a :class:`list`.
 
-   One possible wrinkle is that the iterables passed to :func:`zip` could have
+   One thing to consider is that the iterables passed to :func:`zip` could have
    different lengths; sometimes by design, and sometimes because of a bug in
    the code that prepared these iterables.  Python offers three different
    approaches to dealing with this issue:
 
-   * By default, :func:`zip` has short-circuit behavior.  If any of the
-     iterables are shorter than the rest, :func:`zip` will ignore the remainders
-     of the longer iterables, cutting off the result to the length of the shortest
-     iterable::
+   * By default, :func:`zip` stops when the shortest iterable is exhausted.
+     :func:`zip` will ignore the remaining items in the longer iterables, cutting
```

Comment:
Change "zip" to "it" here, to avoid repetition.

Reply:
Done.